src/lib/relics/relicScorerPotential.ts

```diff
@@ -619,25 +620,27 @@ export class RelicScorer {
       if (relic.substats.length < 4) {
         for (const [stat, weight] of availableSubstats) {
           if (weight >= availableSubstats[availableSubstats.length - 1][1]) {
-            bestAddedStats.push(i18next.t(`common:Stats.${stat}`))
+            bestAddedStats.push(stat)
           }
         }
       }
       const candidateSubstats: [SubStats, number][] = meta.sortedSubstats.filter((x) => relic.main.stat !== x[0])// All substats that could possibly exist on the relic
       const bestUpgradedStats: SubStats[] = [] // Array of all substats possibly on relic sharing highest weight
 
-      const validUpgrades = {
-        ...Utils.arrayToMap(relic.substats, 'stat'),
-        ...Utils.stringArrayToMap(bestAddedStats),
+      const validUpgrades: Record<SubStats, object | true | undefined> = {
```

Comment:
Fine for now but next time we revisit this file this should probably be checked - if a type is `object | true | undefined` that seems like a bug waiting to happen